gitter/tests/test_message_type.py

```diff
@@ -0,0 +1,230 @@
+                     'or it needs more changes? '}
+        m_type2 = message_type(message2s)
+        self.assertEqual(m_type2, 'question')
+
+        message3s = {'text': 'Can I work on this?'}
+        m_type3 = message_type(message3s)
+        self.assertEqual(m_type3, 'question')
+
+        message4s = {'text': 'Can someone assign to me?'}
+        m_type4 = message_type(message4s)
+        self.assertEqual(m_type4, 'question')
+
+        message5s = {'text': 'How did you put a tickmark on a commit ?'}
+        m_type5 = message_type(message5s)
+        self.assertEqual(m_type5, 'question')
+
+        message6s = {'text': 'what to do after rebase?'}
+        m_type6 = message_type(message6s)
+        self.assertEqual(m_type6, 'question')
+
+        message7s = {'text': 'how to Use github interface for review?'}
+        m_type7 = message_type(message7s)
+        self.assertEqual(m_type7, 'question')
+
+        message8s = {'text': 'But dont i have to wait for my previous '
+                     'PR to get merged?'}
+        m_type8 = message_type(message8s)
+        self.assertEqual(m_type8, 'question')
+
+        message9s = {'text': 'is there a separate bears room?'}
+        m_type9 = message_type(message9s)
+        self.assertEqual(m_type9, 'question')
+
+        message10s = {
+            'text': 'What does invoking with a single option mean?'}
+        m_type10 = message_type(message10s)
+        self.assertEqual(m_type10, 'question')
+
+        message11s = {'text': 'Any hint in which file should i add this test?'}
+        m_type11 = message_type(message11s)
+        self.assertEqual(m_type11, 'question')
+
+        message12s = {'text': 'Should i click on the create pull request?'}
+        m_type12 = message_type(message12s)
+        self.assertEqual(m_type12, 'question')
+
+        message13s = {'text': 'is the bot down?'}
+        m_type13 = message_type(message13s)
+        self.assertEqual(m_type13, 'question')
+
+        message14s = {'text': 'can someone tell me which is the docstring '
+                      'file in -utils'}
+        m_type14 = message_type(message14s)
+        self.assertEqual(m_type14, 'question')
+
+        message15s = {'text': 'I have done a newcomer issue, and also '
+                      'reviewed a newcomer issue, What furthere '
+                      'steps for becoming a developer?'}
+        m_type15 = message_type(message15s)
+        self.assertEqual(m_type15, 'question')
+
+        message16s = {
+            'text': 'ok, can you give me some more time to work on this?'}
+        m_type16 = message_type(message16s)
+        self.assertEqual(m_type16, 'question')
+
+        message17s = {'text': 'I had a question why is corobo down?'}
+        m_type17 = message_type(message17s)
+        self.assertEqual(m_type17, 'question')
+
+        message18s = {'text': 'ok, so what should I do now?'}
+        m_type18 = message_type(message18s)
+        self.assertEqual(m_type18, 'question')
```

Comment:
create your own assertion method  assertIsQuestion('ok, so what should I do now?')